spec/support/responses.rb

```diff
@@ -0,0 +1,8 @@
+class Responses
+  class << self
+    def mock_faraday(code, headers, body)
+      env = Faraday::Env.new(status: code, response_headers: headers, response_body: body)
```

Comment:
this uses the `Struct.new` syntax [introduced in ruby 3.2](https://bugs.ruby-lang.org/issues/16806), so it needs to be refactored to use the old style for compatibility with older rubies

Reply:
yeah, i was poking around, looked like it was because of how i was initializing for the unit test.